txpool/txpool.go

```diff
@@ -208,6 +232,17 @@ func NewTxPool(
 		pool.topic = topic
 	}
 
+	// initialize and populate deployment whitelist
+	pool.deploymentWhitelist = deploymentWhitelist{
+		addresses: map[string]bool{},
+	}
+
+	if config.DeploymentWhitelist != nil {
+		for _, addr := range config.DeploymentWhitelist {
+			pool.deploymentWhitelist.add(addr)
+		}
+	}
+
```

Comment:
Please consider making this an init helper in the txpool, it's cluttered to be in the `NewTxPool` function

Reply:
fixed b3b933fc544c709c057e1455967f716afa1cde44

Reply:
Much cleaner, thank you 🙏

Reply:
general go question.
When creating functions for initialization of some struct. Should it be a method of that struct or helper?